mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/core/ExecuteStageRequest.java

```diff
@@ -47,6 +57,7 @@
     private List<Parameter> parameters = new LinkedList<Parameter>();
     private SchedulingInfo schedulingInfo;
     private MantisJobDurationType durationType;
+    private Optional<String> nameOfJobProviderClass;
```

Comment:
Documentation.

Reply:
+1. Why jobProviderClass is in the request now? do we ignore the jobProviderClass inside the zip?

Reply:
@calvin681 no this is just for testing purposes. Imagine running a job that's defined in the same codebase. In that case, the only definition that's needed is the job provider class.